book/src/rust-client/mint_consume_send_tutorial.md

```diff
@@ -0,0 +1,450 @@
+# Mint, Consume, and Send Assets
+
+*Using the Miden Client in Rust to Mint, Consume, and Send Assets*
```

Comment:
nit: can you say here (and on other places) Miden client instead of Miden Client (the same for Miden node)